spec/features/sessions_spec.rb

```diff
@@ -3,24 +3,20 @@
 describe 'Sessions' do
   subject { page }
 
-  before do
+  let(:password) { 'rspec_pass' }
+  let(:user) do
     create(
-      :configuration,
-      name: 'admin:password',
-      value: ::BCrypt::Password.create('rspec_pass')
-    )
-    @user = create(
       :user,
       :author,
       password_hash: ::BCrypt::Password.create('rspec_pass')
     )
   end
 
-  let(:password) { 'rspec_pass' }
-
-  let(:login) do
+  # This needs to be a helper and not a let() block, because let is memoized
+  # and reused.
+  def login
```

Comment:
You're not wrong but maybe memoized was the intent. 

Running this code is costly, and redundant if it's done before hundreds of other specs. If we want a login helper I'd go the route of faking the session as logged in instead of performing log in actions.

Unless you can firmly say this has no negative affect on the total run time I'd say I'm strongly against.

Reply:
Agreed on all grounds, but again pro-parity (we need multiple users there, and can't be memoized).

I thought about using [`login_as_user` from ControllerMacros](https://github.com/dradis/dradis-ce/blob/b81de648c088ab4b371ade481eae126e9fa70831/spec/support/controller_macros.rb#L7), but then I realised this is a feature spec, so it's prolly the one place where filling in the form makes sense.